js/documentation.js

```diff
@@ -0,0 +1,50 @@
+//Collapse presentations
+$("#presentation").click(function () {
+    $("#video").collapse('hide');
+    $("#pdf").collapse('show');
+
+});
+//Collapse videos
+$("#videos").click(function () {
+    $("#pdf").collapse('hide');
+    $("#video").collapse('show');
+});
+
+//Collapse pdf
+$('#pdf-list li').click(function () {
+    let id = $(this).find('a').attr('id');
```

Comment:
```suggestion
    const id = $(this).find('a').attr('id');
```

Reply:
Make sure to use const whenever the instance assignation does not change, this helps to other programmers to understand faster the code.